marlowe-dashboard-client/src/Contract/Types.purs

```diff
@@ -56,37 +51,12 @@ data PreviousStepState
   = TransactionStep TransactionInput
   | TimeoutStep Slot
 
--- MarloweParams are used to identify a Marlowe contract in the PAB, and the wallet
--- companion contract state is a Map MarloweParams MarloweData. We are not currently
--- generating PureScript types to match the Haskell MarloweParams and MarloweData
--- types (side note: perhaps we should be). We have a CurrencySymbol type in
--- Marlowe.Semantics, but it is just an alias for String. We could use that here for
--- the `rolesCurrency` value, and convert using its Bridge instance when sharing data
--- with the PAB. However, we currently don't need to do anything with MarloweParams on
--- the frontend except save them and send them back to the PAB (to join a contract that
--- is already running), so it is simpler just to use the generated type in this case.
```

Comment:
This is a useful comment. I'd re-add it in the new place.

Reply:
:+1: